src/components/questions/QuestionForm.vue

```diff
@@ -39,8 +40,12 @@ async function onSubmit() {
   if (!isFormCorrect) return;
 
   questionData.value.id = uuidv4();
-  console.log(questionData.value);
-  // TODO:send candidateData to mockAPI, test api call
+  try {
+    const store = useQuestionStore();
+    store.postQuestion(questionData.value);
```

Comment:
it seems there supposed to be `await`

Reply:
+ while the data is being saved, the controls might need to be disabled

Reply:
do we need to update the questions list after this request. I can't check it at the moment since the list doesn't work (items there are not expanded)